cvat-canvas/src/typescript/drawHandler.ts

```diff
@@ -467,11 +473,13 @@ export class DrawHandlerImpl implements DrawHandler {
         };
 
         this.canvas.on('mousedown.draw', (e: MouseEvent): void => {
-            if (initialPoint.x === null || initialPoint.y === null) {
-                const translated = translateToSVG(this.canvas.node as any as SVGSVGElement, [e.clientX, e.clientY]);
-                [initialPoint.x, initialPoint.y] = translated;
-            } else {
-                this.drawInstance.fire('drawstop');
+            if (e.button === 0 && !e.altKey) {
```

Comment:
Why alt key here?

Reply:
As for other shapes, when we hold alt, points are not set when clicking canvas (so, you may drag the canvas in drawing mode, for example)